docs/examples/helm/impress.values.yaml

```diff
@@ -114,60 +130,47 @@ yProvider:
     tag: "latest"
 
   envVars:
+    COLLABORATION_BACKEND_BASE_URL: https://docs.127.0.0.1.nip.io
     COLLABORATION_LOGGING: true
-    COLLABORATION_SERVER_ORIGIN: https://impress.127.0.0.1.nip.io
+    COLLABORATION_SERVER_ORIGIN: https://docs.127.0.0.1.nip.io
     COLLABORATION_SERVER_SECRET: my-secret
     Y_PROVIDER_API_KEY: my-secret
-    COLLABORATION_BACKEND_BASE_URL: https://impress.127.0.0.1.nip.io
-    NODE_EXTRA_CA_CERTS: /usr/local/share/ca-certificates/cacert.pem
-
-  # Mount the certificate so yProvider can establish tls with the backend
-  extraVolumeMounts:
-    - name: certs
-      mountPath: /usr/local/share/ca-certificates/cacert.pem
-      subPath: cacert.pem
-
-  extraVolumes:
-    - name: certs
-      configMap:
-        name: certifi
-        items:
-        - key: cacert.pem
-          path: cacert.pem
-
-posthog:
-  ingress:
-    enabled: false
-  ingressAssets:
-    enabled: false
 
 ingress:
   enabled: true
-  host: impress.127.0.0.1.nip.io
+  host: docs.127.0.0.1.nip.io
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 10m
```

Comment:
Could be better to increase, I think uploading bigger Word documents is very likely to happen.